test/bdd/fixtures/demo/.env

```diff
@@ -13,9 +13,9 @@ CONSENT_LOGIN_SERVER_IMAGE=docker.pkg.github.com/trustbloc/edge-sandbox/login-co
 
 # Edge agent
 USER_AGENT_IMAGE=docker.pkg.github.com/trustbloc-cicd/snapshot/wallet-web
-USER_AGENT_IMAGE_TAG=0.1.5-snapshot-50fdbe2
+USER_AGENT_IMAGE_TAG=0.1.5-snapshot-2cea370
 USER_AGENT_SUPPORT_IMAGE=docker.pkg.github.com/trustbloc-cicd/snapshot/wallet-server
-USER_AGENT_SUPPORT_IMAGE_tag=0.1.5-snapshot-50fdbe2
+USER_AGENT_SUPPORT_IMAGE_tag=0.1.5-snapshot-2cea370
```

Comment:
@talwinder50 Did you test this change ? Do you have the latest code from edge-sandbox ?

The latest wallet and web version starts from 0.1.6-snapshot-xyz - https://github.com/trustbloc-cicd/snapshot/packages/514386?version=0.1.6-snapshot-2cea370

Reply:
updated